### ISSUE TITLE
Errors thrown for template type

### ISSUE THREAD

**Issue by ADmad** [slevomat/coding-standard#1012]
Version: 6.3.5

For e.g. code like below using inline annotation with templated type

```php
        /** @psalm-var TObject $instance */
        $instance = $this->_create($className, $name, $config);
```
generates error like
```
 Class name \Foo\Bar\TObject in @psalm-var should be referenced via a fully qualified name.
(SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
```

I expected no error to be generated.

Refs #954

**Comment by kukulich** [comment]
@ADmad How is the template defined?

**Comment by ADmad** [comment]
Using `@template TObject` in docblock for the class.

**Comment by ADmad** [comment]
[This](https://github.com/cakephp/cakephp/blob/a39e054746dddeba8af634c0b131cd7388e39ae3/src/Core/ObjectRegistry.php#L107) is the exact file in question. Error is shown only for that specific inline comment, not any other docblocks.

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/ba467b4d64a527e05a7d7c2defbce10b664bf9c5

**Comment by ADmad** [comment]
Thank you.
